src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader/AbstractReaderTests.java

```diff
@@ -123,7 +122,7 @@ protected String createShardStatMetrics(long indexingThrottleTime,
             long storedFieldsMemory, long termVectorsMemory,
             long normsMemory, long pointsMemory, long docValuesMemory,
             long indexWriterMemory, long versionMapMemory,
-            long bitsetMemory, FailureCondition condition) {
+            long bitsetMemory, FailureCondition condition, long shardSizeInBytes) {
```

Comment:
nit: Re-order the params, `shardSizeInBytes` can be placed before the `condition`

Reply:
Done